app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    commands_list = command.split()
+    copy_command = commands_list[0]
+    old_file = commands_list[1]
+    new_file = commands_list[2]
```

Comment:
You should check that `commands_list` contains 3 elements before unpacking values